src/Components/AffectedClustersTable/AffectedClustersTable.js

```diff
@@ -280,7 +289,7 @@ const AffectedClustersTable = ({ query, rule, afterDisableFn }) => {
   };
 
   return (
-    <div id="affected-list-table">
+    <div id="affected-list-table" data-ouia-safe={!loadingState}>
```

Comment:
@Fewwy, loadingState means there is an animation/table update/loading entries and so on... Logically, if we tell the table is "safe," this will be the opposite of this flag

Reply:
Ok thanks for the clarification